### PR TITLE
offline_fs_kbs: Add a sample aa-offline_fs_kbc-keys.json file

### LINKED ISSUE #231: Add a sample aa-offline_fs_kbc-keys.json file for offline_fs_kbc
We provide a sample aa-offline_fs_kbc-keys.json file, although we describe the  aa-offline_fs_kbc-keys.json in README file but it's better provide a sample aa-offline_fs_kbc-keys.json file too.

### PR DESCRIPTION
**Pull request by liudalibj**
fix https://github.com/confidential-containers/guest-components/issues/231

Signed-off-by: Da Li Liu <liudali@cn.ibm.com>

### CODE REVIEW

**Comment by liudalibj** [comment]
@Xynnn007 @jialez0 please help to review, this PR just update the README file and add a sample aa-offline_fs_kbc-keys.json file to offline_fs_kbc.

**Comment by Xynnn007** [comment]
I think you mean `aa-offline_fs_kbc-keys.json` not `aa-offline_fs_kbc-resources.json`? 

Besides, the `aa-offline_fs_kbc-keys.json` being reviewed is [already used for image-rs' integration test](https://github.com/confidential-containers/image-rs/blob/main/test_data/offline-fs-kbc/aa-offline_fs_kbc-keys.json), so there will not be any potential compatibility issues.

**Comment by liudalibj** [comment]
> Thanks for the example file.
> 
> I think you mean `aa-offline_fs_kbc-keys.json` not `aa-offline_fs_kbc-resources.json`?
> 
> Besides, the `aa-offline_fs_kbc-keys.json` being reviewed is [already used for image-rs' integration test](https://github.com/confidential-containers/image-rs/blob/main/test_data/offline-fs-kbc/aa-offline_fs_kbc-keys.json), so there will not be any potential compatibility issues.

My mistake, updated the title and commit message, Thanks!

File: src/kbc_modules/offline_fs_kbc/README.md [hunk 1]
```diff
@@ -11,13 +11,17 @@ The guest must provide OpenSSL at runtime.
...
-    "key_id1": "cGFzc3BocmFzZXdoaWNobmVlZHN0b2JlMzJieXRlcyE=",
+    "key_id1": "base64-encoded-key",
+    "key_id2": "base64-encoded-key",
     ...
 }
 ```
 
 with the 32-byte keys base64-encoded.
 
+The script [generate_keys.sh](https://github.com/confidential-containers/attestation-agent/blob/main/sample_keyprovider/src/enc_mods/offline_fs_kbs/generate_keys.sh) can be used to generate some sample keys based on `/dev/random` if it fits your use case sufficiently well.
```
Comment (jodh-intel) [inline src/kbc_modules/offline_fs_kbc/README.md:22]: Nit: You might want to use backticks in the link names for the two filenames you are linking too for consistency with the `/dev/random` file. That would also be consistent with Kata's doc standards (https://github.com/kata-containers/kata-containers/blob/main/docs/Documentation-Requirements.md#files-and-command-names).

For example:

```suggestion
The script [`generate_keys.sh`](https://github.com/confidential-containers/attestation-agent/blob/main/sample_keyprovider/src/enc_mods/offline_fs_kbs/generate_keys.sh) can be used to generate some sample keys based on `/dev/random` if it fits your use case sufficiently well.
```

**Review by jodh-intel — APPROVED** [review]
Thanks @liudalibj. One nit, but overall...

lgtm

File: src/kbc_modules/offline_fs_kbc/README.md (hunk 1 above)
Comment (liudalibj) [reply src/kbc_modules/offline_fs_kbc/README.md]: thanks @jodh-intel I updated the filenames with between two backticks

**Comment by jodh-intel** [comment]
Thanks @liudalibj !

**Review by jialez0 — APPROVED** [review]

**Pull request merged**